plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -130,73 +125,115 @@ export const Transitional = ( {
 							window.open( homeUrl, '_blank' );
 						} }
 					>
-						{ __( 'Preview store', 'woocommerce' ) }
+						{ __( 'View store', 'woocommerce' ) }
```

Comment:
There are some tests relying on this string that are failing.

Reply:
Good call! Tests updated & took the opportunity to add a new one :) : https://github.com/woocommerce/woocommerce/pull/45546/commits/c103c1129b4deedd97d7ca1f7980655a8ffef3b4